Python 3.13.2 (tags/v3.13.2:4f8bb39, Feb  4 2025, 15:23:48) [MSC v.1942 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license()" for more information.

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/palindrome.py
True
False
True

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/palindrome.py
False
False
False

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/palindrome.py
True
False
True

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py
Traceback (most recent call last):
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py", line 10, in <module>
    print(find_largest(numbers))
NameError: name 'numbers' is not defined. Did you forget to import 'numbers'?

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py
20

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py
20

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py
20

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py
25

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/findlargestnumber.py
25

= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py
Traceback (most recent call last):
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py", line 4, in <module>
    print(count_vowels("Hello World"))
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py", line 3, in count_vowels
    return sum (1 for char in s if char in vowles)
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py", line 3, in <genexpr>
    return sum (1 for char in s if char in vowles)
NameError: name 'vowles' is not defined
>>> 
= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py
Traceback (most recent call last):
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py", line 4, in <module>
    print(count_vowels("Hello World"))
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py", line 3, in count_vowels
    return sum (1 for char in s if char in vowles)
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py", line 3, in <genexpr>
    return sum (1 for char in s if char in vowles)
NameError: name 'vowles' is not defined
>>> 
= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py
1
1
1
>>> 
= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py
1
1
1
>>> 
= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py
0
0
0
>>> 
= RESTART: C:/Users/Admin/AppData/Local/Programs/Python/Python313/countvowles.py
3
1
6
